src/plugins/condition/ConditionManager.js

```diff
@@ -103,12 +103,16 @@ export default class ConditionManager extends EventEmitter {
 
     createCondition(conditionConfiguration) {
```

Comment:
I don't think we need to make a copy of the configuration here

Reply:
Cloning really means creating a condition with a configuration that is a deep copy of the source condition. I had been doing that deep copy step in createCondition(). Instead, I moved that operation to cloneCondition() and passed the deep copy to createCondition(). I think this makes it clearer what's happening.